src/index.html

```diff
@@ -22,6 +22,431 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        class="header__link"
+        href="images/logo.png"
+      >
+        <img
+          src="images/logo.png"
+          alt="logo"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active">Apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link">Smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              data-qa="nav-hover"
+              class="nav__link"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link">Gadgets</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link">Tablets</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link">Photo</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <article
+        data-qa="card"
+        class="card"
+      >
+        <div class="card__image">
+          <a href="#">
+            <img
+              class="card__image-content"
+              src="images/imac.jpeg"
+              alt="content"
+            />
+          </a>
+        </div>
+        <h3 class="card__info">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h3>
+        <p class="card__code">Product code: 195434</p>
+        <div class="reviews">
+          <div class="stars stars--4">
```

Comment:
The 'stars' block should follow BEM methodology where each star is an element of the block (e.g., 'stars__star'). If 'stars--4' is a modifier indicating a rating of 4, ensure that the styles are applied correctly to reflect this state.